<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::QuickResponseCodeDialog</class>
 <widget class="QDialog" name="Otter::QuickResponseCodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QR Code</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Otter::QuickResponseCodeWidget" name="quickResponseCodeWidget" native="true"/>
   </item>
   <item>
    <widget class="QLineEdit" name="messageLineEdit">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::QuickResponseCodeWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/QuickResponseCodeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
